.eslintrc

```diff
@@ -0,0 +1,63 @@
+{
+    "extends": "semistandard",
+    "rules": {
+        "indent": [2, 4, {"SwitchCase": 1}],
+        "space-before-function-paren": [2, {"anonymous": "always", "named": "never"}],
+        "one-var": 0,
+        "no-throw-literal": 1,
+        "backbone/collection-model": 1,
+        "backbone/model-defaults": 1,
+        "backbone/no-model-attributes": 1,
+        "backbone/no-view-onoff-binding": 1,
+        "backbone/no-view-collection-models": 1,
+        "underscore/collection-return": 2,
+        "underscore/matches-shorthand": [2, "always"],
+        "underscore/no-unnecessary-bind": 2,
+        "underscore/prefer-compact": 2,
+        "underscore/prefer-constant": 2,
+        "underscore/prefer-filter": 2,
+        "underscore/prefer-findwhere": 2,
+        "underscore/prefer-invoke": 2,
+        "underscore/prefer-map": 2,
+        "underscore/prefer-matches": 2,
+        "underscore/prefer-noop": 1,
+        "underscore/prefer-pluck": 2,
+        "underscore/prefer-reject": 2,
+        "underscore/prefer-times": 2,
+        "underscore/prefer-underscore-method": 1,
+        "underscore/prefer-underscore-typecheck": 1,
+        "underscore/prefer-where": 2,
+        "underscore/preferred-alias": 2,
+        "underscore/prop-shorthand": [2, "always"]
+    },
+    "env": {
+        "browser": true,
+        "jquery": true
+    },
+    "plugins": [
+        "backbone",
+        "underscore"
+    ],
+    "globals": {
+        "_": true,
+        "girder": true,
+        "Backbone": true,
+        "marked": true,
+        "JSONEditor": true,
+        "sprintf": true
+    },
+    "settings": {
+        "backbone": {
+            "Collection": [
+                "girder.Collection"
+            ],
+            "Model": [
+                "girder.Model"
+            ],
+            "View": [
+                "girder.View",
+                "girder.views.MetadatumEditWidget"
```

Comment:
That is how the plugin detects if the object is a view or not.  By default, it will only check against objects that follow this pattern:
```
Backbone.View.extend({...})
```
This makes it also check these cases
```
girder.View.extend({...})
girder.views.MetadatumEditWidget.extend({...})
```
I didn't add any other views because we never use them as base objects.